content/Build and Grow Cross Company Knowledge.md

```diff
@@ -0,0 +1,39 @@
+![Share Knowledge](images/knowledge_share.jpg)
+
+# Build and grow cross-company knowledge
+
+*Culture* is the way a company does things, its processes and values, and how it generates outcomes. It's never easy to build, share, and promote knowledge across a medium-sized organization. That task requires leadership, rules, a strong culture, and having effective systems in place.
+
+I've always been passionate about how knowledge sharing has a multiplier effect on the quality of what each person can deliver. I've seen junior developers, after a just few weeks, deliver higher quality work than what I could have produced years ago - even in a nonchallenging environment and even after years of experience.
+
+## We value shared knowledge
+
+When a new technology emerges, we allocate senior talent to research and transmit this knowledge across the organization. We communicate it through talks, meetups, and specific Slack channels. Over time, the complexity of the discussion starts to rise. We don't just cover specific questions. Our whole company brainstorms around each new engineering problem.
+
+## Problem solving
+
+I'm not a fan of using only specific programming languages or technologies. A fantastic technology applied to the wrong problem can create an even more significant problem. An engineer needs to know many different technologies, their weaknesses, and how to apply the right solution.
+
+When your only tool is a hammer, you'll treat every problem as if it were a nail. You don't need an in-depth knowledge of everything, and you don't need to be a "Jack of all trades." But a basic understanding of different industry technologies is mandatory for a great engineer.
```

Comment:
`"Jack of all trades."` I think the point should be outside of the quotation marks scope